File: AplicacionPeliculas/Espera.cs
using System;
using System.Windows.Forms;

namespace AplicacionPeliculas
{
    public partial class Espera
    {
        public Espera()
        {
            InitializeComponent();
        }
        private void Espera_Load(object sender, EventArgs e)
        {
            SimularProgreso();
            StartPosition = FormStartPosition.CenterScreen;
            FormBorderStyle = FormBorderStyle.None;
            ProgressBar1.Maximum = 100;

        }

        public void SimularProgreso()
        {
            for (int i = 0; i <= 100; i++)
            {
                ProgressBar1.Value = i;
                Application.DoEvents();
                System.Threading.Thread.Sleep(100);
            }

            if (ProgressBar1.Value == 100)
            {
                My.MyProject.Forms.Peliculas.Show();
                Close();
            }
            ProgressBar1.Value = 0;
        }

        private void Button1_Click(object sender, EventArgs e)
        {

        }
    }
}

File: AplicacionPeliculas/Espera.vb
Public Class Espera
    Private Sub Espera_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        SimularProgreso()
        Me.StartPosition = FormStartPosition.CenterScreen
        Me.FormBorderStyle = FormBorderStyle.None
        ProgressBar1.Maximum = 100

    End Sub

    Public Sub SimularProgreso()
        For i As Integer = 0 To 100
            ProgressBar1.Value = i
            Application.DoEvents()
            System.Threading.Thread.Sleep(100)
        Next

        If ProgressBar1.Value = 100 Then
            Peliculas.Show()
            Me.Close()
        End If
        ProgressBar1.Value = 0
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs)

    End Sub
End Class